app/models/attestation_template.rb

```diff
@@ -142,4 +141,12 @@ def replace_tags_with_values_from_data(text, tags, data)
       text
     end
   end
+
+  def to_string(champ)
```

Comment:
Pour que ce soit futur-proof, je propose :
```ruby
if champ.value.present?
  case champ.type_champ
  when 'date'
    Date.parse(champ.value).strftime('%d/%m/%Y')
  end
else
  ""
end
```

Reply:
D'ailleurs, tu veux pas mettre ça dans un helper histoire que je puisse itérer là-dessus et utiliser le helper dans l'affichage des champs dans l'onglet "Demande" ?

Reply:
On peut utiliser un helper coté view et coté back ?

Reply:
Je pense : https://stackoverflow.com/questions/5130150/rails-how-to-use-a-helper-inside-a-controller

Sinon méthode d'instance sur `Champ` ?